src/App/utils/SelectPanel/index.js

```diff
@@ -148,6 +146,12 @@ class SelectPanel extends Component {
                                         </nav>
                                     }
                                 </li>)}
+                                <li className={"main-nav-li"}>
```

Comment:
what is the reason behind -in general in this PR- removing the utilities route from the dynamic routes, and hardcoding it here instead?

Reply:
Yes. The sidebar is completely rebuilded when going to the page, based on html attributes. In this file we are "scanning" for routes, and then building the sidebar tree based on the incoming routes. 
I tried to add a utilities route file to manually give the titles to the searchbar for example.. but because of the code snippet I mentioned above, everything broke since we are (again) scanning for routes.. 
It is easier (or maybe not) to explain when I show it to you, or we can huddle it!

Reply:
let's create a new ticket to improve the search functionality
resolving this conversation